.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Team
+# @manh-t is the Team Lead
+* @manh-t @doannimble @luongvo @markgravity
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```

Comment:
This file contains errors due to invalid user @ducbm051291 
I think we can update this quick